Source/Core/Core/Movie.cpp

```diff
@@ -1161,7 +1185,26 @@ void PlayController(GCPadStatus* PadStatus, int controllerID)
   }
 
   memcpy(&s_padState, &s_temp_input[s_currentByte], sizeof(ControllerState));
-  s_currentByte += sizeof(ControllerState);
+<<<<<<< HEAD
```

Comment:
Seems like an incomplete rebase occurred here

Reply:
You would be correct... Fixed and force pushing.